airflow/providers/airbyte/sensors/airbyte.py

```diff
@@ -79,3 +81,59 @@ def poke(self, context: Context) -> bool:
 
         self.log.info("Waiting for job %s to complete.", self.airbyte_job_id)
         return False
+
+
+class AirbyteJobSensorAsync(AirbyteJobSensor):
```

Comment:
WDYT about adding a new param `deferrable` to the existing `AirbyteJobSensor` instead of introducing a new sensor?

Reply:
@pankajastro I combined the Deferable Sensor and the Orignal Sensor using the method you outlined above. While I was at it, I also added the `deferrable` parameter to the operator as well. Let me know what you think.